include/SDL3/SDL_video.h

```diff
@@ -52,11 +52,12 @@ extern "C" {
  */
 typedef struct
 {
-    Uint32 format;              /**< pixel format */
-    int w;                      /**< width, in screen coordinates */
-    int h;                      /**< height, in screen coordinates */
-    int refresh_rate;           /**< refresh rate (or zero for unspecified) */
-    void *driverdata;           /**< driver-specific data, initialize to 0 */
+    Uint32 format;                  /**< pixel format */
+    int w;                          /**< width, in screen coordinates */
+    int h;                          /**< height, in screen coordinates */
+    int refresh_rate_numerator;     /**< refresh rate numerator (or zero for unspecified) */
+    int refresh_rate_denominator;   /**< refresh rate denominator (never zero) */
```

Comment:
Personally, I don't like this api because these 2 variables are very intertwined.
How about introducing a `Ratio32`/`Ratio64` struct typedef?

Reply:
so a float / double, this was discussed here ... https://github.com/libsdl-org/SDL/pull/6938

Reply:
I mean a proper struct with numer/denom members.

Reply:
Or use an anonymous struct here.

Reply:
yes maybe. and this should be fairly easy to change

Reply:
Maybe something like this?
```c
typedef SDL_RefreshRate
{
    Uint32 numerator;
    Uint32 denominator;
} SDL_RefreshRate;
```